### PR TITLE
Introduce material app

### PR DESCRIPTION
**Pull request by kasperbirch1**
#### Link to issue
https://reload.atlassian.net/browse/DDFSOEG-175

#### Description

This PR introduce material app.
Material app is container for material page and it receives 2 props(pid, workId) that it gets from url (can also be set via storybook)

#### Screenshot of the result
**With url parameters**
Skærmbillede 2022-07-04 kl  12 56 12
**With storybook**
Skærmbillede 2022-07-04 kl  12 56 39

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

**UTILS FILES ARE CHECKOUT FOM** https://github.com/danskernesdigitalebibliotek/dpl-react/pull/36

### CODE REVIEW

File: src/apps/material/material.dev.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+} as ComponentMeta<typeof MaterialEntry>;
+
+const Template: ComponentStory<typeof MaterialEntry> = (args) => (
+  <MaterialEntry {...args} />
+);
+
+export const withStorybookArgs = Template.bind({});
+withStorybookArgs.args = {
+  pid: `${870970}-basis:${46336461}`,
```
Comment (spaceo) [inline src/apps/material/material.dev.tsx:26]: Is there a particular reason why you use template literals here?

File: src/apps/material/material.entry.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { getUrlQueryParam } from "../../core/utils/helpers";
+// import { withText } from "../../core/utils/text";
```
Comment (spaceo) [inline src/apps/material/material.entry.tsx:3]: You need to choose wether this import should be here or not ;)

File: src/apps/material/material.entry.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+export interface MaterialEntryProps {
+  pid: Pid;
+  workId: WorkID;
+}
+
+const MaterialEntry: React.FC<MaterialEntryProps> = ({
+  pid: attrpid,
+  workId: attrWorkId
+}) => {
+  // If a q string has been defined as a data attribute use that
```
Comment (spaceo) [inline src/apps/material/material.entry.tsx:16]: Please update comment

**Review by spaceo — CHANGES_REQUESTED** [review]

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Review by sinejespersen — APPROVED** [review]

**Pull request merged**